recipes/abinit/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "abinit" %}
+{% set version = "8.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b362bda0ccefbc7254d653d551ba1952aa4a92f9f845208caa3446cb8960ad76
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - make
```

You need ``autoconf`` as well
```suggestion
    - make
    - autoconf
```